### ISSUE TITLE
request : single script to do all steps (diag, clim, export)

### ISSUE THREAD

**Issue by etiennesky** [brodeau/barakuda#8]
Hi it would be nice to have a single script which does all the 3 steps
- diag
- climatology
- export to html

I am thinking of a wrapper script which takes as arguments the RUN as well as Y1 and Y2 and any other optional arguments and passes them on to barakuda.sh and build_clim.sh. Or is this too complicated to do?

Also in this manner any temporary files downloaded (in the case of autosubmit MMA files) could be downloaded only once.
